modules/contacts/client/components/ContactPresentational.js

```diff
@@ -25,7 +25,7 @@ export default function ContactPresentational({
         <div>
           <i className="icon-fw icon-building text-muted"></i>
           <small>
-            <Trans ns="contact" locationLiving={locationLiving}>
+            <Trans ns="contacts" locationLiving={locationLiving}>
```

Comment:
Passing in `t` seems neater than re-specifying the namespace.

Reply:
I see. I agree with that.
There is what i think is a bug in babel-plugin-i18next-extract. It doesn't pick up the proper namespace from the passed `t` (at least for me). The translation strings from `Trans` without explicit `ns` end up in `translation.json` instead of their proper `namespace.json`. I've opened gilbsgilbs/babel-plugin-i18next-extract#153 and #1368.
Should we document the reasons for using `ns` instead of `t` more clearly? Add TODOs everywhere? Add `t={t}`?
Alternatively, it's possible that I'm overlooking something and the plugin works as it should.

Reply:
Added `t={t}` and TODO about removing `ns` to all `Trans` components. (squashed into the first commit)
I think that's all we can do with this issue now.